FF1Lib/FF1Rom.cs

```diff
@@ -345,7 +346,7 @@ public void Randomize(Blob seed, Flags flags, Preferences preferences)
 			// Put this before other encounter / trap tile edits.
 			if ((bool)flags.AllowUnsafeMelmond)
 			{
-				EnableMelmondGhetto(flags.RandomizeFormationEnemizer);
+				EnableMelmondGhetto(flags.RandomizeFormationEnemizer || flags.RandomizeEnemizer);

```

Comment:
Instead of passing this here - can we make the method that adjusts all the formations and separate it out. Something like ```void updateEncounterTiles() { ... }``` and just call it at the top of this method like we do with `ShiftEarthOrbDown()`

Then everything else after can assume the rewritten data and we don't need any checks like this.

Reply:
If that's not clear, I mean the function that changes all the `0x0A80` to `0x0A00` everywhere so that `0x0A00` is now the only way random encounters will work.

Reply:
I suggested that and nitz said he did it.  I could write such a function and update everything that uses it.  However I still need to pass the flag to MelmondGhetto because I write my own formations rather than the hardcoded ones the original function uses.